src/main/java/io/mosip/mimoto/controller/CredentialsController.java

```diff
@@ -0,0 +1,94 @@
+package io.mosip.mimoto.controller;
+
+import io.mosip.mimoto.core.http.ResponseWrapper;
+import io.mosip.mimoto.dto.ErrorDTO;
+import io.mosip.mimoto.dto.IssuerDTO;
+import io.mosip.mimoto.dto.idp.TokenResponseDTO;
+import io.mosip.mimoto.dto.mimoto.CredentialIssuerWellKnownResponse;
+import io.mosip.mimoto.dto.mimoto.CredentialsSupportedResponse;
+import io.mosip.mimoto.dto.mimoto.VCCredentialRequest;
+import io.mosip.mimoto.dto.mimoto.VCCredentialResponse;
+import io.mosip.mimoto.exception.ApiNotAccessibleException;
+import io.mosip.mimoto.exception.IdpException;
+import io.mosip.mimoto.service.CredentialService;
+import io.mosip.mimoto.service.IdpService;
+import io.mosip.mimoto.service.IssuersService;
+import io.mosip.mimoto.util.DateUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.*;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static io.mosip.mimoto.exception.PlatformErrorMessages.API_NOT_ACCESSIBLE_EXCEPTION;
+import static io.mosip.mimoto.exception.PlatformErrorMessages.MIMOTO_PDF_SIGN_EXCEPTION;
+
+@RestController
+@RequestMapping(value="/credentials")
+public class CredentialsController {
```

Comment:
rename `CredentialsController` to `CredentialController`
and endpoint from `/credentials` to `/credential`

Reply:
According to the REST standards, Resources are usually referred in plurals 
ref : https://restfulapi.net/resource-naming/

**"Use the “plural” name to denote the collection resource archetype."**